src/Compilers/CSharp/Portable/BoundTree/BoundSwitchExpression.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal partial class BoundSwitchExpression
+    {
+        public BoundDecisionDag DecisionDag { get; }
```

Comment:
>DecisionDag [](start = 32, length = 11)

Can `DecisionDag` be added to definition in `BoundNodes.xml`?

Reply:
`DecisionDag` cannot be added to the definition in `BoundNodes.xml` because the generated visitors would take exponential time due to revisiting dag nodes with more than one parent.

Reply:
The `<Field/>` could be marked `SkipInVisitor="true"`.

---
In reply to: [173013517](https://github.com/dotnet/roslyn/pull/24059#discussion_r173013517) [](ancestors = 173013517)

Reply:
I don't see any `SkipInVisitor` used anywhere in C#'s `BoundNodes.xml`. Are you sure that works?

---
In reply to: [173016288](https://github.com/dotnet/roslyn/pull/24059#discussion_r173016288) [](ancestors = 173016288,173013517)

Reply:
`SkipInVisitor` works. I'm using it in a pending PR.

Reply:
I like the idea of making this change but I'd prefer to defer it to a separate PR.

---
In reply to: [173243694](https://github.com/dotnet/roslyn/pull/24059#discussion_r173243694) [](ancestors = 173243694)

Reply:
See [https://github.com/dotnet/roslyn/projects/13#card-8006567](https://github.com/dotnet/roslyn/projects/13#card-8006567) for the backlog item.

---
In reply to: [173252375](https://github.com/dotnet/roslyn/pull/24059#discussion_r173252375) [](ancestors = 173252375,173243694)